### ISSUE TITLE
升级到 weasel-0.14.3 版本后，发现五笔86打不出"势" ; z 后接拼音无效

### ISSUE THREAD

**Issue by Zenoe** [rime/home#394]
1."势" 用五笔86打不出来
2. z 后接拼音无效

操作系统: windows 7

**Comment by Ace-Who** [comment]
从什么版本升上来的？

**Comment by Zenoe** [comment]
从0.10.0.0升级上来的

**Comment by Ace-Who** [comment]
同样条件下没有复现。

重新部署再试试。

**Comment by Zenoe** [comment]
我把原来的配置目录C:\Users\xxx\AppData\Roaming\Rime 删除，重新安装0.14.3版本后，我发现默认没有五笔86。接着想通过在"输入法设定"增加五笔86，可是在“方案名称”里找不到。是不是不支持五笔86了呢？（ps：不是五筆畫）

**Comment by Ace-Who** [comment]
不是。[自 0.10.0 起，不再预置大多数方案](https://rime.im/release/weasel/)。可自行[從 Rime 輸入方案倉庫](https://github.com/rime/plum)[獲取輸入方案](https://github.com/rime/home/wiki/UserGuide)。具体做法是：

> Windows 用家可以通過 小狼毫 0.11 以上「輸入法設定／獲取更多輸入方案」調用配置管理器。

五笔 86 方案的包名为 [wubi](https://github.com/rime/rime-wubi)。
